syntax = "proto2";

package qcraft.planner;
option go_package = "qcraft-proto.ai/onboard/planner/proto/expert_planner_debug";

import "onboard/planner/proto/trajectory_validation.proto";
import "onboard/planner/speed/proto/speed_finder.proto";
import "onboard/planner/scheduler/proto/scheduler.proto";
import "onboard/planner/decision/proto/constraint.proto";

message ExpertPlannerDebugProto {
  optional SchedulerOutputProto scheduler = 1;
  optional ConstraintProto constraint = 2;
  optional SpeedFinderDebugProto speed_finder = 3;
  optional TrajectoryValidationResultProto traj_validation = 4;
}